app/models/operatingsystems/rhcos.rb

```diff
@@ -0,0 +1,47 @@
+class Rhcos < Operatingsystem
+  #
+  # Example PXE URLs:
+  # http://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.5/4.5.6/rhcos-installer-kernel-x86_64
+  # http://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.5/4.5.6/rhcos-installer-initramfs.x86_64.img
+  #
+  PXEFILES = {
+    kernel: 'rhcos-installer-kernel-$arch',
```

Comment:
broken with openshift 4.6.. because of a rename..

https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.6/4.6.1/rhcos-live-initramfs.x86_64.img

i think its good that it was not merged yet!